app/scripts/directives/fromFile.js

```diff
@@ -157,6 +157,10 @@ angular.module("openshiftConsole")
               openTemplateProcessModal();
             // Else if any resources already exist
             } else if (!_.isEmpty($scope.updateResources)) {
+              // If the updated resource is a Template, set the process option to false
+              if ($scope.updateResources.length === 1 && $scope.updateResources[0].kind === "Template") {
+                $scope.templateOptions.process = false;
```

Comment:
this doesn't seem right.  if I checked both the Save and Process options then it should ask me to confirm the update and then go to the from template page